handlers/main.yml

```diff
@@ -1,9 +1,13 @@
 ---
+# Use manage.sh so we use the same env vars from .env that we use on
+# other invocations of django manage.py. Unfortunately that means we cannot
+# use the ansible django_manage module.
+# TODO: see if we could use an ansible lookup plugin to read the .env file
+# and pass the values to the environment configuration item?
 - name: collectstatic
-  django_manage:
```

Comment:
Ah interesting, so this means manage.py doesn't require dotenv?

Reply:
manage.sh uses dotenv when invoking manage.py